LayoutTests/fast/repaint/table-section-visual-overflow.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<style type="text/css">
+    #test
+    {
+        display: table-row-group;
+        outline-color: black;
+        outline-style: solid;
+        outline-width: 10px;
+    }
+    #row
+    {
+        display: table-row;
+    }
+    #table
+    {
+        display: table;
+        table-layout: fixed;
+        /* Position the table to overlap a raster tile only for outline */
+        position: relative;
+        left: 145px;
+    }
+    #cell
+    {
+        display: table-cell;
+        height: 100px;
+        width: 100px;
+    }
+</style>
+<script src="../../fast/repaint/resources/text-based-repaint.js"></script>
+<div id="table">
+    <div id="test">
+        <div id="row">
+            <div id="cell"></div>
+        </div>
+    </div>
+</div>
+<script>
+window.enablePixelTesting = true;
```

Comment:
This does nothing in webkit.

Reply:
I checked few test cases and it seems that they do have 'enablePixelTesting' in them, hence, I kept it. Let me remove it. Thanks!

Reply:
Removed. Thanks!